.github/workflows/linters.yml

```diff
@@ -0,0 +1,62 @@
+name: Linters
```

Comment:
- Fix your Pull Request Linters issue, try Installing the right files provided [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js).

- Fix issues described in your **PR** before to request another code review

Reply:
Thanks but it's exactly the same I'm using